### DISCUSSION TITLE
Error after Frigate LXC script default installation - OpenVino model file /openvino-model/ssdlite_mobilenet_v2.xml not found

### DISCUSSION THREAD

**Discussion by supershopping** [tteck/Proxmox#3542] [Help Needed] [answered]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

Hardware: Lenovo ThinkCentre M900 Tiny CPU i5-6500T - 8GB ram
Software: Proxmox 8.2.2

Install frigate LXC with below script on default settings.
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/frigate.sh)"
Once I save and restart my frigate config, getting error from frigate system log.

Tag
frigate.detectors.plugins.openvino
Message
OpenVino model file /openvino-model/ssdlite_mobilenet_v2.xml not found.

Full log screenshot attached.

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 11

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

<img width="1288" alt="Screenshot 2024-08-18 at 10 18 00 AM" src="https://github.com/user-attachments/assets/db8701bf-a42b-42d9-a180-cc36ab3676e3">
<img width="1301" alt="Screenshot 2024-08-18 at 10 18 53 AM" src="https://github.com/user-attachments/assets/845c151f-4b1f-4b7b-b494-4d1c93e5900e">

**Comment by tteck** [comment, marked as answer]
Could your config be wrong?
https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/frigate-install.sh#L126

**Reply by supershopping** [reply]
Thanks for pionting out my mistake. I just moved frigate from HAOS to LXC and using the same config cause the trouble. 
Was using "path: /openvino-model/ssdlite_mobilenet_v2.xml" before.

**Reply by oerix** [reply]
> Could your config be wrong?
> 
> https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/frigate-install.sh#L126

I've updated to this path, but still get 'not found' message in logs. Do I need to install some dependencies?

**Reply by supershopping** [reply]
No. If you keep all script default settings, it should run without any issue. If only thing you modify is the cofig, then the issue most likely is related to your config.

**Reply by oerix** [reply]
> No. If you keep all script default settings, it should run without any issue. If only thing you modify is the cofig, then the issue most likely is related to your config.

Not really. My config is set as default, the only line I updated is the OpenVino path in question, as the log suggested.

```
OpenVino model file /openvino-model/FP16/ssdlite_mobilenet_v2.xml not found.
"unknown: File /opt/frigate/frigate/detectors/plugins/openvino.py", line 41, in __init__"
```

@tteck Has anything changed with the recent 0.14 update, or must something else be defined?

**Comment by letsruletheworld** [comment]
did you get it resoled ?  i stuck at the same problem.

**Reply by theluks** [reply]
This works:

```
path: /openvino-model/FP16/ssdlite_mobilenet_v2.xml
labelmap_path: /openvino-model/coco_91cl_bkgr.txt
```
